cluster/clientbin.sh

```diff
@@ -93,9 +93,13 @@ function get_bin() {
   # The bazel go rules place binaries in subtrees like
   # "bazel-bin/source/path/linux_amd64_pure_stripped/binaryname", so make sure
   # the platform name is matched in the path.
-  locations+=($(find "${KUBE_ROOT}/bazel-bin/${srcdir}" -type f -executable \
-    -path "*/${host_os}_${host_arch}*/${bin}" 2>/dev/null || true) )
-  echo $( (ls -t "${locations[@]}" 2>/dev/null || true) | head -1 )
+  locations=()
+  while IFS=$'\n' read -r line; do
+    locations+=( "${line}" )
+  done < <(find "${KUBE_ROOT}/bazel-bin/${srcdir}" -type f -executable \
+    -path "*/${host_os}_${host_arch}*/${bin}" 2>/dev/null || true)
+  
```

Comment:
change reason:
```
In ./cluster/clientbin.sh line 96:
  locations+=($(find "${KUBE_ROOT}/bazel-bin/${srcdir}" -type f -executable \
              ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```